### PR TITLE
added accessToken option

### PR DESCRIPTION
**Pull request by Sitocp**
Thank you for submitting a pull request!

## Description

Added option for accessToken.

## Type of Pull Request

- [X] Adding a feature
- [ ] Fixing a bug
- [ ] Maintaining documents
- [ ] Others ()

## Verify the followings

- [X] Code is up-to-date with the `main` branch
- [x] No lint errors after `npm run lint`
- [ ] Make sure all the exsiting features working well

### CODE REVIEW

File: lib/map-generator.ts [hunk 1]
```diff
@@ -110,13 +112,15 @@ export default class MapGenerator {
     dpi: number = 300,
     format:string = Format.PNG.toString(),
     unit: Unit = Unit.mm,
+    accessToken?: string,
```
Comment (JinIgarashi) [inline lib/map-generator.ts]: There is an error. Could you fix it?

**Review by JinIgarashi — CHANGES_REQUESTED** [review]
@Sitocp Thank you for pull request. The change looks fine. However there is an error in CI, could you fix it? so I can merge and release it.

**Review by JinIgarashi — APPROVED** [review]
Thank you for fixing quickly!

**Comment by Sitocp** [comment]
Thank you! ^^

**Comment by JinIgarashi** [comment]
@Sitocp now [v1.2.9](https://github.com/watergis/mapbox-gl-export/releases/tag/v1.2.9) was released! thanks

**Pull request merged**